apps/client/src/app/modules/freecompany-workshops/import-workshop-from-pcap-popup/import-workshop-from-pcap-popup.component.html

```diff
@@ -0,0 +1,34 @@
+<nz-spin [nzSpinning]="(dataLoaded$ | async) && (isLoading$ | async)">
+  <div fxLayout="column" fxLayoutGap="10px">
+    <div fxLayoutGap="10px" *ngIf="freecompany$ | async as freecompanyData">
+      <div>
+        <h3>Free Company</h3>
+        <ng-container *ngTemplateOutlet="freecompanyTpl;context:{fc: freecompanyData}"></ng-container>
+      </div>
+      <div>
+        <ng-container *ngIf="airshipList$ | async as airshipsData">
+          <h3>Airships</h3>
+          <ng-container *ngTemplateOutlet="vesselTpl;context:{vessels: airshipsData}"></ng-container>
+        </ng-container>
+      </div>
+      <div>
+        <ng-container *ngIf="submarineList$ | async as submarinesData">
+          <h3>Submersibles</h3>
+          <ng-container *ngTemplateOutlet="vesselTpl;context:{vessels: submarinesData}"></ng-container>
+        </ng-container>
+      </div>
+    </div>
+    <nz-alert nzType="info" *ngIf="!(dataLoaded$ | async)" [nzMessage]="'FREECOMPANY_WORKSHOPS.Import_from_pcap_notice' | translate"></nz-alert>
+    <button *ngIf="(dataLoaded$ | async) && !(isLoading$ | async)" (click)="save()" nz-button nzType="primary">Save</button>
+  </div>
+</nz-spin>
+<ng-template #freecompanyTpl let-fc="fc">
+  <div>Server: {{ fc?.server }}</div>
+  <div>FC: {{ fc?.name }}</div>
+  <div>Rank: {{ fc?.rank }}</div>
```

Comment:
Would be nice to translate these, most of them are already translated inside the files, feel free to reuse existing keys if the context matches.